IntrepidSwiftWisdom.podspec

```diff
@@ -17,6 +17,6 @@ Pod::Spec.new do |s|
 
   s.subspec "Core" do |cs|
     cs.source_files = "SwiftWisdom/Core/**/**/*.swift"
-    cs.dependency 'IP-UIKit-Wisdom', '0.0.9'
+    cs.dependency 'IP-UIKit-Wisdom', '0.0.10'
```

Comment:
no harm to bump both if you're updating. however, none of the changes will be visible until we make a push with the proper tags.